Adapter.H

```diff
@@ -93,6 +93,9 @@ private:
         //- Switch to enable the FluidStructureInteraction module
         bool FSIenabled_ = false;
 
+        //- Switch for faceTriangulation (nearest projection)
+        bool provideMeshConnectivity_ = false;
```

Comment:
For shorter, rename `provideMeshConnectivity` to `meshConnectivity` everywhere. It also makes more sense when using it in `write()` and `read()`.

Reply:
This is done.